<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arma3 Mod ID Extractor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>591</width>
      <height>224</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="mdpckbtn">
       <property name="text">
        <string>Select modpack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mdpckPath">
       <property name="text">
        <string>Path for the modpack will be displayed here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outbtn">
       <property name="text">
        <string>Select output file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outPath">
       <property name="text">
        <string>Path for the output file will be displayed here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extractBtn">
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="addComaCbx">
       <property name="text">
        <string>Add comma between ID's</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="oneLineCbx">
       <property name="text">
        <string>Save all ID's in one line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>App made by capt-jack00 on Github using Qt6 and Python.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
